Day5/index.js

```diff
@@ -0,0 +1,202 @@
+    let tempS = parseFloat( $('#hsl-s-value').val() );
+    let tempL = parseFloat( $('#hsl-l-value').val() );
+    if( (tempL >= 0) && (tempL <= 1) ) {
+      let myArray = hslToRgb(tempH, tempS, tempL);
+      r = Math.floor( myArray[0] );
+      g = Math.floor( myArray[1] );
+      b = Math.floor( myArray[2] );
+      displayRGB( myArray );
+      let myArray2 = rgbToHsv(r, g, b);
+      displayHSV(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The l value of hsl must be [0, 1] value.');
+    }
+  } );
+
+  $('#hsv-h-value').on( 'input' , () => {
+    let tempH = parseFloat( $('#hsv-h-value').val() );
+    let tempS = parseFloat( $('#hsv-s-value').val() );
+    let tempV = parseFloat( $('#hsv-v-value').val() );
+    if( (tempH >= 0) && (tempH <= 1) ) {
+      let myArray = hsvToRgb(tempH, tempS, tempV);
+      r = Math.floor( myArray[0] );
+      g = Math.floor( myArray[1] );
+      b = Math.floor( myArray[2] );
+      displayRGB( myArray );
+      let myArray2 = rgbToHsl(r, g, b);
+      displayHSL(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The h value of hsv must be [0, 1] value.');
+    }
+  } );
+
+  $('#hsv-s-value').on( 'input' , () => {
+    let tempH = parseFloat( $('#hsv-h-value').val() );
+    let tempS = parseFloat( $('#hsv-s-value').val() );
+    let tempV = parseFloat( $('#hsv-v-value').val() );
+    if( (tempS >= 0) && (tempS <= 1) ) {
+      let myArray = hsvToRgb(tempH, tempS, tempV);
+      r = Math.floor( myArray[0] );
+      g = Math.floor( myArray[1] );
+      b = Math.floor( myArray[2] );
+      displayRGB( myArray );
+      let myArray2 = rgbToHsl(r, g, b);
+      displayHSL(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The s value of hsv must be [0, 1] value.');
+    }
+  } );
+
+  $('#hsv-v-value').on( 'input' , () => {
+    let tempH = parseFloat( $('#hsv-h-value').val() );
+    let tempS = parseFloat( $('#hsv-s-value').val() );
+    let tempV = parseFloat( $('#hsv-v-value').val() );
+    if( (tempV >= 0) && (tempV <= 1) ) {
+      let myArray = hsvToRgb(tempH, tempS, tempV);
+      r = Math.floor( myArray[0] );
+      g = Math.floor( myArray[1] );
+      b = Math.floor( myArray[2] );
+      displayRGB( myArray );
+      let myArray2 = rgbToHsl(r, g, b);
+      displayHSL(myArray2);
+      $('#color-field').css( 'background-color' , `rgb(${r},${g},${b})` );
+    }
+    else {
+      alert('The v value of hsv must be [0, 1] value.');
+    }
+  } );
+
+  $('#clear-butt').on( 'click' , () => {
+    r = 0;
+    g = 0;
+    b = 0;
+    $('#r-value').val(0);
+    $('#g-value').val(0);
+    $('#b-value').val(0);
+    $('#hsl-h-value').val(0)
+    $('#hsl-s-value').val(0)
+    $('#hsl-l-value').val(0)
+    $('#hsv-h-value').val(0)
+    $('#hsv-s-value').val(0)
+    $('#hsv-v-value').val(0)
```

Comment:
Try using jQuery's goodies. You can query for all the input at once and change their values.
```
$('input').val(0); // query for all <input> tag and set value to 0
```